pkg/controllers/management/clusterprovisioner/provisioner.go

```diff
@@ -460,6 +460,10 @@ func (p *Provisioner) reconcileRKENodes(clusterName string) ([]v3.RKEConfigNode,
 			continue
 		}
 
+		if !v3.NodeConditionDeployed.IsTrue(machine) && !isCustom(machine) {
```

Comment:
I don't like this condition.  Let me think about it

Reply:
agentDeployed? :)

Reply:
AgentDeployed is fine